<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data_view_window</class>
 <widget class="QMainWindow" name="Data_view_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="output_text">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>1181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1181</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="RegisterDataPoints">
     <attribute name="title">
      <string>Register Data Points</string>
     </attribute>
     <widget class="QListWidget" name="dataPointName_listWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>371</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="add_name_button">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Register Data Point</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="input_name_text">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Preview:</string>
      </property>
     </widget>
     <widget class="QLabel" name="preview_txt_label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_names_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>510</y>
        <width>371</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear List</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Table_View_Tab">
     <attribute name="title">
      <string>Table View</string>
     </attribute>
     <widget class="QTableView" name="data_tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1161</width>
        <height>681</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="auto_scroll_checkBox">
      <property name="geometry">
       <rect>
        <x>1074</x>
        <y>700</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto Scroll</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_data_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>700</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Save as CSV</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
